### PR TITLE
upgrading operator-sdk to 0.16.0

### PR DESCRIPTION
**Pull request by jackyalbo**

### CODE REVIEW

**Comment by liranmauda** [comment]
Is this working with go v1.12 ?

**Comment by jackyalbo** [comment]
go 1.13 as with 0.13.0

File: go.mod [hunk 1]
```diff
@@ -40,6 +38,7 @@ require (
 
 /// Pinned to kubernetes-1.16.2
 replace (
+	github.com/docker/docker => github.com/moby/moby v0.7.3-0.20190826074503-38ab9da00309
```
Comment (guymguym) [inline go.mod:41]: Where is this coming from? operator-sdk?

Comment (jackyalbo) [reply go.mod]: changed to docker to make it clearer

**Comment by jackyalbo** [comment]
> Oh forgot to edit the travis CI script that installs the operator-sdk version:
> https://github.com/noobaa/noobaa-operator/blob/master/.travis/install-operator-sdk.sh

ok fixed

**Pull request merged**